### DISCUSSION TITLE
Is there any way to use generators to generate types only?

### DISCUSSION THREAD

**Discussion by async3619** [prisma/prisma#2590] [Q&A] [answered]
```prisma
// This is your Prisma schema file,
// learn more about it in the docs: https://pris.ly/d/prisma-schema

datasource db {
  provider = "mysql"
  url      = env("DATABASE_URL")
}

generator client {
  provider  = "prisma-client-js"
  output    = "__generated__"
}

model User {
  id      Int      @default(autoincrement()) @id
  email   String   @unique
  name    String?
}
```

when I use `prisma generate`, It will generate quite many files in `__generated__` directory. but I really want to generate **types only** like `graphql-code-generator`.

Is there any way to do it?

**Comment by ryands17** [comment, marked as answer]
Hey @async3619 :wave: 
Currently there's no way just to generate the types only as `prisma generate` will generate the other files as well. Could you create a feature request [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) stating the above use-case that you have?

**Comment by ludralph** [comment]
Hi @async3619,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
